app/controllers/settings/identity_proofs_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class Settings::IdentityProofsController < Settings::BaseController
+  layout 'admin'
+
+  before_action :authenticate_user!
+  before_action :check_required_params, only: :new
+
+  def index
+    @proofs = AccountIdentityProof.where(account: current_account).order(provider: :asc, provider_username: :asc)
+    @proofs.each(&:refresh!)
```

Comment:
@xgess Is this necessary? n synchronous HTTP requests on page load is not great. I would think the worker queued after the proof is saved would take care of checking if the proof is live

Reply:
The `refresh!` do not seem to be synchronous (it spawns a worker?), but I question the need to trigger a refresh each time that page is visited.

Reply:
It does `worker_class.new.perform(@proof)` which *is* synchronous

Reply:
the main reason i had to do this was that there's nothing currently built on the mastodon side to recognize remotely revoked proofs. i thought it might be weird if a user revokes a proof in keybase, then days later still sees it as live in mastodon until the first refresh. we had it as a note to talk about / build something that might let keybase inform mastodon a proof is revoked, or build a rake task for mastodon to check and invalidate revoked proofs. 

i'm flexible on this though. if you want to change it to be async, i think that's reasonable.